devel/ci/integration/bodhi/Dockerfile-f34

```diff
@@ -1,4 +1,4 @@
-FROM bodhi-ci/f32
+FROM bodhi-ci/f34
```

Comment:
You will need to change
```
RUN python3 setup.py build && pip3 install --no-use-pep517 . && pip3 install koji
```
with
```
RUN python3 setup.py bdist_wheel && pip3 install ./dist/*.whl && pip3 install koji
```
to fix f34-integration-build

Reply:
Oh, also add python3-wheel to dnf install command, see https://github.com/fedora-infra/bodhi/pull/4195/commits/6f5d36debbbca31d7b394308a17217015946766a

Reply:
Done and done. f34-integration-build  works now, thanks!

Reply:
You're welcome.

For the f32-* tests which are always required, I think there is a setting at project level in github that can only be changed by an admin... unfortunately, removing them from mergify config isn't enough.

Reply:
Found that setting as well. You think this looks good enough? I'd really like to merge this and then start looking at some of  the stale PR's we have around the place, with the work of fixing and refactoring the integration tests to be postponed to a later date.